crates/daphne-server/src/lib.rs

```diff
@@ -84,6 +94,8 @@ pub struct App {
     /// colleciton requests. Note that in a production Leader, it is necessary to store this state
     /// across requsets.
     test_leader_state: Arc<Mutex<InMemoryLeaderState>>,
+    #[cfg(feature = "test-utils")]
+    interop_bearer_tokens: crate::roles::InteropBearerTokens,
```

Comment:
What do you mean by "interop" bearer token? Note that the type has no documentation and I can't tell what this is from context.

Reply:
it is indeed to support the interop draft (https://divergentdave.github.io/draft-dcook-ppm-dap-interop-test-design/draft-dcook-ppm-dap-interop-test-design.html), will document